scripts/pre_commit_linter.py

```diff
@@ -202,6 +202,15 @@
             'core/tests/performance_framework/perf_domain.py'),
         'excluded_dirs': ('scripts/',)
     },
+    {
+        'regexp': r'# pylint: disable=[A-Z][0-9]{4}',
+        'message': "Please remove pylint exculsion if it is unnecessary or "
```

Comment:
Change the message to `Please remove this pylint pragma if it is...`

Reply:
Done